features/gezag-minderjarige/1.2-minderjarig-niet-overleden.feature

```diff
@@ -0,0 +1,296 @@
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+      Voorbeelden:
+      | geboortedatum          | omschrijving                                                          |
+      | dit jaar - 18 jaar     | alleen jaar is bekend en persoon wordt of is dit jaar 18 jaar oud     |
+      | deze maand - 18 jaar   | jaar en maand is bekend en persoon wordt of is deze maand 18 jaar oud |
+      | dit jaar - 19 jaar     | alleen jaar is bekend en persoon werd vorig jaar 18 jaar oud          |
+      | vorige maand - 18 jaar | jaar en maand is bekend en persoon werd vorige maand 18 jaar oud      |
+
+
+  Regel: Wanneer er onderzoek loopt naar de geboortedatum is gezag niet te bepalen
+
+    Abstract Scenario: persoon is volgens geboortedatum <meer of minderjarig> en <onderzoek omschrijving> staat in onderzoek
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is <meer of minderjarig>
+      * heeft de volgende onderzoek gegevens
+      | aanduiding in onderzoek (83.10) |
+      | <aanduiding in onderzoek>       |
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                                          |
+      | type        | GezagNietTeBepalen                                                                              |
+      | toelichting | gezag is niet te bepalen omdat de volgende relevante gegevens in onderzoek staan: <toelichting> |
+
+      Voorbeelden:
+      | meer of minderjarig | aanduiding in onderzoek | onderzoek omschrijving | toelichting                                            |
+      | minderjarig         | 010000                  | hele categorie persoon | geboortedatum van de persoon, geboorteland van persoon |
+      | minderjarig         | 010300                  | hele groep geboorte    | geboortedatum van de persoon, geboorteland van persoon |
+      | minderjarig         | 010310                  | geboortedatum          | geboortedatum van de persoon                           |
+      | meerderjarig        | 010310                  | geboortedatum          | geboortedatum van de persoon                           |
```

Comment:
Iedereen met een geboortedatum in onderzoek, zelfs als een persoon 50 jaar oud is, krijgt gezag niet te bepalen? 
Ik zou verwachten dat als iemand meerderjarig is, geen gezag wordt geretourneerd. Als we dit al willen doen, dan lijkt me dit alleen opportuun voor personen met een leeftijd net boven de 18 oid.